tools/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualOutputWriter.scala

```diff
@@ -435,8 +441,8 @@ object QualOutputWriter {
       ToolUtils.formatDoublePrecision(sumInfo.estimatedGpuTimeSaved) ->
         ESTIMATED_GPU_TIMESAVED.size,
       sumInfo.recommendation -> SPEEDUP_BUCKET_STR_SIZE,
-      sumInfo.unsupportedExecs -> UNSUPPORTED_EXECS.size,
-      sumInfo.unsupportedExprs -> UNSUPPORTED_EXPRS.size
+      sumInfo.unsupportedExecs -> UNSUPPORTED_EXECS_MAX_SIZE,
```

Comment:
this is always going t5o use the max size, it should either be max or the size of the header string if its empty, so in the previous function you need to calculate that.

Reply:
done. Added function to calculate the correct length